src/Http/Resources/Json/MakesAnonymousPaginationResultAwareResourceCollection.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Lampager\Laravel\Http\Resources\Json;
+
+/**
+ * Trait MakesAnonymousPaginationResultAwareResourceCollection
+ *
+ * @mixin \Illuminate\Http\Resources\Json\JsonResource
+ */
+trait MakesAnonymousPaginationResultAwareResourceCollection
+{
+    /**
+     * Create new anonymous resource collection.
```

Comment:
```suggestion
     * Create a new anonymous resource collection.
```

Reply:
Descriptions are just copied from Laravel core 🤣

Reply:
Fun fact that they did right in the other place 😄  :
https://github.com/illuminate/http/blob/07138d2851763a4f2c185b72a3c72e913bf57c5d/Resources/Json/AnonymousResourceCollection.php#L15